### PR TITLE
Extend GRPC ModelRepositoryParameter to allow bytes

### PR DESCRIPTION
**Pull request by GuanLuo**

### CODE REVIEW

File: protobuf/grpc_service.proto [hunk 1]
```diff
@@ -1142,6 +1142,12 @@ message ModelRepositoryParameter
     //@@       A string parameter value.
     //@@
     string string_param = 3;
+
```
Comment (tanmayv25) [inline protobuf/grpc_service.proto:1145]: You must also add this to kfserve copy here: https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/grpc_predict_v2.proto#L276

Comment (GuanLuo) [reply protobuf/grpc_service.proto:1145]: I don't think it is needed, the kfserve one only covers the basic protocol (infer), whereas this is the Triton specific extension. The protobuf message is added within Triton domain

Comment (rmccorm4) [reply protobuf/grpc_service.proto:1145]: I agree with Guan. If this was `InferParameter`, it should be updated in KServer. But `ModelRepositoryParameter` is Triton-specific

**Review by rmccorm4 — APPROVED** [review]

File: protobuf/grpc_service.proto (hunk 1 above)
Comment (tanmayv25) [reply protobuf/grpc_service.proto:1145]: Ahh.. That's correct. Missed that point.

**Review by tanmayv25 — APPROVED** [review]

**Pull request merged**